dbt_sugar/core/task/doc.py

```diff
@@ -11,8 +19,125 @@ class DocumentationTask(BaseTask):
     """
 
     def __init__(self, flags: FlagParser) -> None:
-        ...
+        super().__init__()
+        self.flags = flags
+
+    def load_dbt_credentials(self) -> Dict[str, str]:
+        """Method to load the DBT profile credentials."""
+        dbt_profile = DbtProfile(
+            project_name="default",
+            target_name="dev",
+        )
+        dbt_profile.read_profile()
+        dbt_credentials = dbt_profile.profile
+        if not dbt_credentials:
+            logger.info("Not able to locate DBT profile.")
+            exit(1)
+        return dbt_credentials
 
     def run(self) -> int:
```

Comment:
bastien, I dont understand why we have to return int in the run command.

Reply:
it's something it tend to do for runners. I usually return 0. Allows to do things like returning non 0 and crash apps. This video got me into the habit of doing that: https://www.youtube.com/watch?v=Kur4Y7j4vac